tools/dds/dds-config/readme.md

```diff
@@ -0,0 +1,54 @@
+# dds-config tool
+
+## Goal
+This tool is used to define a device as a DDS-compatible device and ensure it operates correctly within a DDS environment.  
+It allows users to manage connection priorities, enabling the selection of either DDS or USB as the preferred connection method when both options are available.
+
+## Description
+Typically, the PC which the device needs to be connected to will have only one ethernet port.  
```

Comment:
Where did you get this info from?
@OhadMeir  please help reviewing this readme with me.
IMO we need 1 Eth cable connected to the PC, right?

we do not require 2 eth port we require 1
If the user want to have another port fir internet access its not related to our tool,
IMO we shouldn't even mention it
What do you think?

Reply:
removed that section